packages/rest/src/__tests__/unit/request-body.validator.test.ts

```diff
@@ -83,20 +83,23 @@ describe('validateRequestBody', () => {
     );
 
     // 3. Trigger a validation with `{coerceTypes: false}` with invalid data
-    expect(() =>
-      validateRequestBody(
+    try {
+      await validateRequestBody(
         {
           value: {city: 'San Jose', unit: '123', isOwner: true},
           schema: ADDRESS_SCHEMA,
         },
         aBodySpec(ADDRESS_SCHEMA),
         {},
         {coerceTypes: false},
-      ),
-    ).to.throw(/The request body is invalid/);
+      );
+    } catch (error) {
+      expect(error.message).to.equal(INVALID_MSG);
+      expect(error.code).to.equal('VALIDATION_FAILED');
+    }
```

Comment:
We use `await expect(aPromise).to.be.rejectedWith(...)` style.

Reply:
Done!